components/image/image.spec.ts

```diff
@@ -137,7 +137,7 @@ describe('Placeholder', () => {
     context.placeholder = PLACEHOLDER;
     const image = debugElement.nativeElement.querySelector('img');
     fixture.detectChanges();
-    tick(600);
+    tick(1000);
```

Comment:
If DataURL is used, is `tick(1000)` still necessary?

Reply:
No I don't thinks so, should i revert it in another PR, since this one is merged?

Reply:
Yes of course.

Reply:
Apparently it is needed.
https://dev.azure.com/ng-zorro/NG-ZORRO/_build/results?buildId=7095&view=ms.vss-test-web.build-test-results-tab&runId=21460&resultId=100799&paneView=debug